xml/hwbp_registers.xml

```diff
@@ -189,7 +189,12 @@
         <field name="m" bits="6" access="R/W" reset="0">
             When set, enable this trigger in M mode.
         </field>
-        <field name="0" bits="5" access="R" reset="0" />
+        <field name="hit" bits="5" access="R/W" reset="0">
+            When this trigger matches, this optional bit becomes set. The
+            debugger can use it to see which trigger(s) matched, and is
```

Comment:
what is "the debugger" when this trigger is not owned by Debug Mode, but rather than by M-Mode? Do you mean that in that case M-Mode SW can use and is responsible for clearing it? I'm not sure if that is a consistent use of the term "the debugger" in the spec so far, which I always assumed was the external debugger connected to Debug Module.

Reply:
You're right. The debugger can be external, or local software. I'll rephrase this.